lib/msal-angular/src/msal.guard.spec.ts

```diff
@@ -450,7 +450,7 @@ describe('MsalGuard', () => {
       username: "test"
     }]);
 
-    guard.canLoad()
+    guard.canMatch()
```

Comment:
May i ask you why are we replacing `canLoad()` with `canMatch()`?

Reply:
`canLoad` has been deprecated and replaced with `canMatch`. https://angular.io/api/router/CanLoad

Reply:
Makes sense, thank you!